src/components/diff-view.jsx

```diff
@@ -17,6 +17,7 @@ import SideBySideRawVersions from './side-by-side-raw-versions';
  * @property {string} diffType
  * @property {Version} a
  * @property {Version} b
+ * @property {boolean} removeFormatting
```

Comment:
Don't think that I should make this change because the new component resides in `change-view` alongside `diff-view` instead of inside `diff-view`. Can you double-check?

Reply:
This was because I was thinking the pattern should be what I put here: https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/225#discussion_r193900267

Didn’t realize that wasn’t clear, sorry.

Reply:
Oh no, I wasn't recognizing the pattern you suggested and immediately latched on to extracting the controls I wrote into a new component. A new object for the settings *themselves* makes total sense.